### DISCUSSION TITLE
Adding the new field through entry type page - add newly added field to the top of the list

### DISCUSSION THREAD

**Discussion by piotrpog** [craftcms/cms#15080] [Ideas] [answered]
I have repeatedly noticed that when i add the new field through entry type page, i then waste time to look for newly added field on the list. 
It would be better if newly added field was displayed somewhere immediately visible, ready to be drag and dropped into field layout - so we don't need to scroll through field list to find it.
It could be automatically placed on top of the list, ignoring alphabetical sorting.

**Comment by brandonkelly** [comment]
I don’t think we should automatically place the field in a tab, because we wouldn’t know which tab/position to place it, so there’s a pretty small chance we’d actually be saving you a drag-n-drop operation.

I agree it would be nice if could make it easier to see the newly-added field, though. So as of Craft 5.2, the search input will get auto-populated with the new field’s name, so the list gets automatically filtered to just that field, or close to it. (ed2403541c28a327704199a9b55ff9db2768457e).

**Reply by brandonkelly** [reply]
5.2.0 is out with that change.

**Comment by brandonkelly** [comment, marked as answer]
Craft 5.6.0 was just released, which brings some UI improvements to field layout designers, making it possible for newly-created fields to be added to tabs immediately (#16374) 🎉
